api/golang/core/lib/enclaves/starlark_run_blocking.go

```diff
@@ -54,6 +54,14 @@ func ReadStarlarkRunResponseLineBlocking(starlarkRunResponseLines <-chan *kurtos
 			} else if responseLine.GetError().GetExecutionError() != nil {
 				executionError = responseLine.GetError().GetExecutionError()
 			}
+		} else if responseLine.GetRunFinishedEvent() != nil {
+			runFinishedEvent := responseLine.GetRunFinishedEvent()
+			if runFinishedEvent.GetIsRunSuccessful() {
+				if runFinishedEvent.GetSerializedOutput() != "" {
```

Comment:
Yeah, not sure TBH. It feels off to me to just dump the output object to the script output which already contains a bunch of other stuffs. Maybe we should add an attribute to StarlarkRunResult and store this separately? At the same time we don't know how this will be used. Maybe this entire stringified scriptOutput is dumb in the first place.
🤷

Reply:
Yeah 100%. It's not used that much, we're missing concrete datapoints to make a good decision here so, I'm fine with any solution here.